src/Microsoft.Identity.Web.DownstreamApi/Microsoft.Identity.Web.DownstreamApi.csproj

```diff
@@ -7,6 +7,9 @@
     <EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>
     <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
+	<ItemGroup>
+		<Compile Include="..\Microsoft.Identity.Web.TokenAcquisition\IdHelper.cs" Link="IdHelper.cs" />
```

Comment:
why as a link?

Reply:
To avoid duplicate code. Is there a common package with internals visible to relationship I can move it instead?

Reply:
can probably move to diagnostics.

Reply:
Done.